core/lib/json_api_client_extension/resource_extension.rb

```diff
@@ -0,0 +1,23 @@
+# MonkeyPatching json_api_client, see: https://github.com/chingor13/json_api_client/pull/263
```

Comment:
This is fine for now as a PR is opened upstream.
It's better to get the upstream fixed than depend on a fork